resources/generate_book.py

```diff
@@ -298,13 +298,41 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
             # todo: info about fruit trees
             # todo: listing of various fruit trees
         )),
-        entry('wild_animals', 'Wild Animals', '', pages=(
-            entity('tfc:turtle', 'This is a turtle', 'Turtle'),
-            image('tfc:textures/gui/book/wip.png'),
+        entry('wild_animals', 'Wild Animals', 'tfc:medium_raw_hide', pages=(
             # Wild animals - address both hostile and passive important animals
-            # todo: info about wild animals
-            # todo: predators
-            # todo: passive animals - link to another section about animal husbandry
+            text('The world of TFC is full of animal life. Some animals are here to help, and some are incredibly dangerous. This section is about wild animals. For information on livestock, animals that can give you items you need, see the $(l:mechanics/animal_husbandry)Animal Husbandry$() page.'),
+            text('$(thing)Predators$() are animals that can attack the player. They are either $(thing)nocturnal$(), only hunting at night$(), or $(thing)diurnal$(), only hunting during the day. When they get a kill, they will not hunt for 12 hours. However, attacking a predator will always anger it. Predators have a chance of pinning you to the ground when they attack, if they get up close. Predators have a home territory that they do not like to leave. If you run far enough away, the predator will return home.'),
```

Comment:
```suggestion
            text('$(thing)Predators$() are animals that can attack the player. They are either $(thing)nocturnal$(), only hunting at night, or $(thing)diurnal$(), only hunting during the day. When they get a kill, they will not hunt for hours. However, attacking a predator will always anger it. Predators have a chance of pinning you to the ground when they attack, if they get up close. Predators have a home territory that they do not like to leave. If you run far enough away, the predator will return home.'),
```
- All predators can pin you? Otherwise would say "Some predators can"